include/curl/curl.h

```diff
@@ -1866,6 +1866,23 @@ enum CURL_NETRC_OPTION {
   CURL_NETRC_LAST
 };
 
+#define GET_CURL_SSLVERSION(x) (x & 0xffff)
+#define GET_CURL_SSLVERSION_MAX(x) (x & 0xffff0000)
+#define SET_CURL_SSLVERSION_MAX(x, val) \
+  CURL_SSLVERSION_MAX_##x = (val << 16)
```

Comment:
You are changing a public header file.  All the defines you introduce need to start with the ```CURL``` prefix.